sdk/servicebus/Azure.Messaging.ServiceBus/src/ServiceBusMessage.cs

```diff
@@ -285,9 +285,7 @@ public static ServiceBusMessage CreateFrom(ServiceBusReceivedMessage message)
             var originalBody = message.Body;
             if (!originalBody.IsEmpty)
             {
-                var clonedBody = new byte[originalBody.Length];
-                Array.Copy(originalBody.ToArray(), clonedBody, originalBody.Length);
-                copiedMessage.Body = clonedBody;
+                copiedMessage.Body = originalBody;
```

Comment:
Good catch! I wonder can we get away with just moving the Body set into the object initializer with the rest of the properties and doing away with the IsEmpty check?

Reply:
I think that can work. Can do tomorrow or so

Reply:
Done